canonical/vanilla-framework
# 75/25% Grid Row (`.row--75-25`)

## Done

- Implements `.row--75-25`, `.row--75-25-on-medium`, and `.row--75-25-on-large`
- Adds `.row--25-75-on-medium` example (there was no example of this previously)

Fixes #5234, [WD-13516](https://warthogs.atlassian.net/browse/WD-13516)

This is needed for simplification of #5230 

## QA

- Review new examples
  - [75-25](https://vanilla-framework-5235.demos.haus/docs/examples/patterns/grid/75-25?theme=light)
  - [75-25 on large](https://vanilla-framework-5235.demos.haus/docs/examples/patterns/grid/75-25-responsive-large?theme=light)
  - [75-25 on medium](https://vanilla-framework-5235.demos.haus/docs/examples/patterns/grid/75-25-responsive-medium?theme=light)
  - [75-25 / 50-50 mixed responsive](https://vanilla-framework-5235.demos.haus/docs/examples/patterns/grid/75-25-mixed-responsive?theme=light)
  - [25-75 on medium](https://vanilla-framework-5235.demos.haus/docs/examples/patterns/grid/25-75-responsive-medium?theme=light)
- Review [combined grid example](https://vanilla-framework-5235.demos.haus/docs/examples/patterns/grid/combined?theme=light) (ensure all grid examples are present)
- Verify [25-75-responsive](https://vanilla-framework-5235.demos.haus/docs/examples/patterns/grid/25-75-responsive) redirects to [25-75-responsive-large](https://vanilla-framework-5235.demos.haus/docs/examples/patterns/grid/25-75-responsive-large?theme=light)
- Review [Grid / Common patterns docs](https://vanilla-framework-5235.demos.haus/docs/patterns/grid#common-patterns)
- Review [4.15 release notes](https://vanilla-framework-5235.demos.haus/docs/whats-new)

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix release (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots
Large breakpoint
![image](https://github.com/user-attachments/assets/c7ad03e6-d8a6-475f-93b8-6dbc5401b1d5)

Medium breakpoint
![image](https://github.com/user-attachments/assets/53024e9b-5609-410e-98c2-ed964e0dbfb5)

Small breakpoint
![image](https://github.com/user-attachments/assets/f9d36f74-9f91-412d-9e8b-cb726522a7f6)



[WD-13516]: https://warthogs.atlassian.net/browse/WD-13516?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
